src/masks/currency.js

```diff
@@ -2,7 +2,7 @@ import { isNil } from "lodash";
 
 const CurrencyMask = {
   mask(value) {
-    return(isNil(value) ? null : value.toString().replace(/[^0-9.,]/g, ""));
+    return(isNil(value) ? null : value.toString().replace(/[^0-9.,-]/g, ""));
```

Comment:
When the currency is getting masked we are unintentionally removing the negative amount sign.